flex/validation/parameter.py

```diff
@@ -51,6 +51,8 @@ def type_cast_parameters(parameter_values, parameter_definitions, context):
             parameter_definition = find_parameter(parameter_definitions, name=key)
         except KeyError:
             continue
+        except ValueError:
+            continue
```

Comment:
I'm a bit concerned that this will also catch this exception https://github.com/maroux/flex/blob/114/flex/parameters.py#L29 which isn't ideal.

Ideally I'd like to not have this be silenced alongside the other exception.